pkg/actors/internal/placement.go

```diff
@@ -224,9 +225,15 @@ func (p *ActorPlacement) Stop() {
 }
 
 // WaitUntilPlacementTableIsReady waits until placement table is until table lock is unlocked.
-func (p *ActorPlacement) WaitUntilPlacementTableIsReady() {
-	if p.tableIsBlocked.Load() {
-		<-p.unblockSignal
+func (p *ActorPlacement) WaitUntilPlacementTableIsReady(ctx context.Context) error {
+	if !p.tableIsBlocked.Load() {
+		return nil
+	}
```

Comment:
please add a UT for this scenario.
Also will this code be executed in normal scenario?

Reply:
I am not sure when this would be executed, probably just during init phases. The code was there before, I have just added support for contexts.

Reply:
Added test